dpi-client/src/main/java/no/difi/meldingsutveksling/dpi/client/internal/BusinessCertificateValidator.java

```diff
@@ -0,0 +1,145 @@
+package no.difi.meldingsutveksling.dpi.client.internal;
+
+import no.difi.certvalidator.Validator;
+import no.difi.certvalidator.ValidatorLoader;
+import no.difi.certvalidator.api.CertificateValidationException;
+import no.difi.certvalidator.lang.ValidatorParsingException;
+import no.difi.meldingsutveksling.dpi.client.internal.domain.Mode;
+import no.difi.meldingsutveksling.dpi.client.internal.domain.RecipePath;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.cert.X509Certificate;
+
+public class BusinessCertificateValidator {
```

Comment:
I stedet for å kopiere denne fra virksert, så kunne den kanskje vært flyttet til commons?